app/editor/src/features/content/hooks/useTab.tsx

```diff
@@ -1,23 +1,24 @@
 import React from 'react';
-import { useChannel } from 'store/hooks';
 
 let _window: Window | null = null;
 
-export const useTab = () => {
-  const channel = useChannel<any>({});
+export interface ITabOptions {
+  useOneTab?: boolean;
+}
 
+// TODO: When `useOnTab=true` they get out of sync easily.
+export const useTab = (options: ITabOptions = { useOneTab: false }) => {
   const [tab, setTab] = React.useState<Window | null>(_window);
 
   return React.useCallback(
     (id: number, path: string = '/contents') => {
-      if (!tab || tab.closed) {
+      if (!tab || tab.closed || !options.useOneTab) {
```

Comment:
Defaults to a new tab for each click.  I'll need to figure out how we can do this another way later.